### ISSUE TITLE
Latest NPM release broken

### ISSUE THREAD

**Issue by adambiggs** [vue-bulma/datepicker#21]
I just tried updating to v1.2.6, but now my app fails to build with the following error:

```
This dependency was not found in node_modules:

* vue-bulma-datepicker in ./~/babel-loader/lib!./~/vue-loader/lib/selector.js?type=script&index=0!./path/to/my/component.vue

Did you forget to run npm install --save for it?
```

I had a look inside `node_modules/vue-bulma-datepicker` and it seems like the release is corrupted by  some kind of windows file path issue:

```sh
vue-bulma-datepicker/
  src/
    # empty
  LICENSE
  package.json
  README.md
  src\index.vue
  src\SingleInput.vue
  src\WrapperInput.vue
```

**Comment by adambiggs** [comment]
v1.2.4 is the last release that works. v1.2.5 and v1.2.6 are both broken.

**Comment by fundon** [comment]
Thanks to report.

**Comment by fundon** [comment]
@luventa Do you execute the command `yarn publish` on Windows?

**Comment by adambiggs** [comment]
@fundon I tried installing v1.2.7 that you just published... and that's broken too! 😵

```
error An unexpected error occurred: "https://registry.yarnpkg.com/vue-bulma-datepicker/-/vue-bulma-datepicker-1.2.7.tgz: invalid tar file"
```

I ran into this myself when trying to publish to npm with `$ yarn publish`. [See here for more info](https://github.com/yarnpkg/yarn/issues/1619#issuecomment-271216626).

**Comment by luventa** [comment]
@fundon Yes, after that I cleaned and reinstalled my `node_modules`  of vue admin. And everything work correctly.
@adambiggs could you please check if folder `vue-bulma-datepicker` exists under your `node_modules`, please?

**Comment by luventa** [comment]
@adambiggs I tested with an empty folder with 1.2.7 and it failed. So I just published 1.2.8 with npm this time.
Test passed, it shall works. Seems `yarn publish` is not working for the time being. Thanks!

**Comment by fundon** [comment]
@adambiggs Thanks.
